### PR TITLE
tc-fixed linting errors from existing files

### PR DESCRIPTION
**Pull request by tommyconner96**
Ran Prettier on all files to fix the lint errors that were already in the project before we began work.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "watch:dev": "nodemon",
     "lint": "npx eslint .",
     "lint:fix": "npx eslint --fix .",
-    "format": "npx prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md)\"",
```
Comment (MrT3313) [inline package.json]: was this something you removed for your individual debugging given your issues with `npx`?

**Review by germainep — APPROVED** [review]
That was a great idea. It's great to have everything organized and up to the correct standards

File: README.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ You can find the deployed project at [🚫URL NAME GOES HERE](🚫copy and paste
...
+|     Student 1     |                                                         Aric Repp                                                          |                                                          Tommy Conner                                                          |                                                         Ryan Schueck                                                          |     Student 5     |     Student 6     |                                                      Germaine Pannell                                                       |                                                         Reed Turgeon                                                         |
```
Comment (MrT3313) [inline README.md:15]: oh look at that! it removed all the cleaned our spacing and added back that line! 

not an issue just remember to turn your text wrap off

**Review by MrT3313 — APPROVED** [review]
Thanks for adding back that npx. Looks good!

**Pull request merged**